src/dispatch/incident/models.py

```diff
@@ -239,6 +239,7 @@ class ProjectRead(DispatchBase):
     id: Optional[PrimaryKey]
     name: NameStr
     color: Optional[str]
+    restrict_stable_to: Optional[IncidentPriorityRead] = None
```

Comment:
A minor nit, but I'm not sure `restrict_stable_to` (and it's other forms) really convey whats going on. To me, something like: `stable_priority` describes what it is `priority` and when it's set `stable`.

Reply:
Good suggestion. I like the way that reads better too. Fixed.